route_verification/irr/src/mbrs.rs

```diff
@@ -0,0 +1,56 @@
+use super::*;
+
+/// Gather `member-of` and `mnt-by` expressions.
+pub fn gather_ref(obj: &RPSLObject, set: &mut Map2DStringVec) {
+    let mut member_ofs = Vec::new();
+    let mut mnt_by = Vec::new();
+    for RpslExpr { key, expr } in expressions(lines_continued(obj.body.lines())) {
+        match key.as_str() {
+            "member-of" => member_ofs.extend(split_commas(&expr).map(str::to_string)),
+            "mnt-by" => mnt_by.extend(split_commas(&expr).map(str::to_string)),
+            _ => (),
+        }
+    }
+    if !member_ofs.is_empty() && mnt_by.is_empty() {
+        warn!("AutNum {} with `member-of` without `mnt-by`.", obj.name);
```

Comment:
If `mnt_by` is empty, the following nested loop would not function, so I added a warning.

However, in my testing, this warning does not show up. Therefore, I assume all AutNums with `member-of` has `mnt-by`.